cvat-ui/src/components/global-error-boundary/global-error-boundary.tsx

```diff
@@ -0,0 +1,165 @@
+            error,
+        };
+    }
+
+    public componentDidCatch(error: Error, errorInfo: React.ErrorInfo): void {
+        const { job } = this.props;
+        const parsed = ErrorStackParser.parse(error);
+
+        const logPayload = {
+            filename: parsed[0].fileName,
+            line: parsed[0].lineNumber,
+            message: error.message,
+            column: parsed[0].columnNumber,
+            stack: error.stack,
+            componentStack: errorInfo.componentStack,
+        };
+
+        if (job) {
+            job.logger.log(LogType.sendException, logPayload);
+        } else {
+            logger.log(LogType.sendException, logPayload);
+        }
+    }
+
+    public render(): React.ReactNode {
+        const { restore, job } = this.props;
+        const { hasError, error } = this.state;
+
+        const restoreGlobalState = (): void => {
+            this.setState({
+                error: null,
+                hasError: false,
+            });
+
+            restore();
+        };
+
+        if (hasError && error) {
+            const message = `${error.name}\n${error.message}\n\n${error.stack}`;
+            return (
+                <div className='cvat-global-boundary'>
+                    <Result
+                        status='error'
+                        title='Oops, something went wrong'
+                        subTitle='More likely there are some issues with the tool'
+                    >
+                        <div>
+                            <Paragraph>
+                                <Paragraph strong>What has happened?</Paragraph>
+                                <Paragraph>Program error has just occured</Paragraph>
+                                <Collapse accordion>
+                                    <Collapse.Panel header='Error message' key='errorMessage'>
+                                        <Text type='danger'>
+                                            <TextArea className='cvat-global-boundary-error-field' autoSize value={message} />
+                                        </Text>
+                                    </Collapse.Panel>
+                                </Collapse>
+                            </Paragraph>
+
+                            <Paragraph>
+                                <Text strong>What should I do?</Text>
+                            </Paragraph>
+                            <ul>
+                                <li>
+                                    <Tooltip title='Copied!' trigger='click'>
+                                        {/* eslint-disable-next-line */}
+                                        <a onClick={() => {copy(message)}}> Copy </a>
+                                    </Tooltip>
+                                    the error message to clipboard
+                                </li>
+                                <li>
+                                    Notify your manager or submit the issue directly on
```

Comment:
Maybe it can be useful to set more generic "administrator"

Reply:
Adjusted it